### ISSUE TITLE
[Slim] Printing class labels within eval_image_classifier.py

### ISSUE THREAD

**Issue by osama** [tensorflow/models#697]
## Slim
I am currently experimenting with different models for image classification in Slim. Following the tutorial there, I have fine-tuned a pre-trained `inception_v2_resnet` model and am trying to [evaluate][1] it. I was wondering if there would be any simple way to modify the [`eval_image_classifier.py`][2] script to print the labels of the images that it is classifying? This would help in adapting this script for use with a test set.

I realize that results are saved by default to `/tmp/tfmodel/` but am not sure how to use the files located there. Thanks!

  [1]: https://github.com/tensorflow/models/tree/master/slim#evaluating-performance-of-a-model
  [2]: https://github.com/tensorflow/models/blob/master/slim/eval_image_classifier.py

**Comment by poxvoculi** [comment]
This kind of usage question is better asked on stackoverflow, since it's not a bug report.  I don't have any expertise in this area.  It might be possible to use the Summary mechanism to accomplish what you want.
